src/engine/SCons/Tool/msvc.xml

```diff
@@ -475,4 +475,43 @@ Valid values are '1' or '0'
 </summary>
 </cvar>
 
+<cvar name="VSWHERE">
+<summary>
+<para>
+Specify the location of vswhere.exe. 
+</para>
+
+<para>
+Propose: The <command>vswhere</command> command is distributed with Microsoft Visual Studio and Build
+    Tools since the 2017 edition, but is also available standalone. It fully detects 2017 and later editions,
+    but with the <option>-legacy</option> argument, can detect installations of the 2010 through 2015
+    editions with limited data returned.
+If &cv-VSWHERE; is set, SCons will use that location.
+</para>
+<para>
+    Otherwise SCons will look in the following locations and set &cv-VSWHERE; to the path of the first <command>vswhere.exe</command>
+located.
+</para>
+
+<itemizedlist>
+<listitem><literal>%ProgramFiles(x86)%\Microsoft Visual Studio\Installer</literal></listitem>
+<listitem><literal>"%ProgramFiles%\Microsoft Visual Studio\Installer</literal></listitem>
+<listitem><literal>%ChocolateyInstall%\bin</literal></listitem>
+</itemizedlist>
+
+<para>
+Note that &cv-VSWHERE; must be set at the same time or prior to the msvc tool being initialized. So either set it as follows
+<literal>Environment(VSWHERE='c:/my/path/to/vswhere')</literal> or 
+if your Environment() is created specifying no tools <literal>tools=[]</literal> or with a list of tools not including msvs or msvc or mslink
+    such as <literal>Environment(Tools=[])</literal>
```

Comment:
this sentence still doesn't seem to have an ending: "...either set... or if ... or with..." then what?